libs/ui/widgets/src/lib/progress-bar/progress-bar.component.html

```diff
@@ -1,12 +1,39 @@
-<div class="flex h-full {{ color.outerBar }} rounded-t-lg rounded-b-lg">
-  <div
-    [style.width.%]="progress"
-    class="flex {{
-      color.innerBar
-    }} my-1 mx-1 transition-width duration-500 ease-in-out rounded-t-md rounded-b-md shadow-xl"
-  >
-    <div class="flex items-center pl-2 py-1 {{ color.text }} font-bold text-4">
-      {{ progress }}%
+<ng-container [ngSwitch]="type">
+  <!-- Light Theme -->
+  <ng-container *ngSwitchCase="'light'">
+    <div class="relative">
```

Comment:
Not a big deal but, is this div really necessary or is it just to add `relative` class? maybe `relative` can be moved in the div below and this div can be removed.